tests/unit/django_spanner/models.py

```diff
@@ -0,0 +1,64 @@
+"""
```

Comment:
Test files also need the copyright header

Reply:
added copyright header.